nuxeo-drive-core/src/main/java/org/nuxeo/drive/adapter/impl/AbstractDocumentBackedFileSystemItem.java

```diff
@@ -90,11 +90,8 @@ protected AbstractDocumentBackedFileSystemItem(String factoryName, DocumentModel
                 parentDoc = docSession.getDocument(parentDocRef);
             }
         } catch (DocumentSecurityException e) {
-            if (log.isDebugEnabled()) {
-                log.debug(String.format(
-                        "User %s has no READ access on parent of document %s (%s), will throw RootlessItemException.",
-                        principal.getName(), doc.getPathAsString(), doc.getId()));
-            }
+            log.debug("User {} has no READ access on parent of document {} ({}), will throw RootlessItemException.",
+                    principal.getName(), doc.getPathAsString(), doc.getId());
```

Comment:
Please use lambdas instead of always calling the getters. If a value is not a constant then a lambda should be used.